ag-docs-wafv2/azuredeploy.json

```diff
@@ -1,291 +1,401 @@
+            "apiVersion": "2019-11-01",
+            "name": "[concat(variables('publicIPAddress'), copyIndex())]",
+            "location": "[parameters('location')]",
+            "sku": {
+                "name": "Standard"
+            },
+            "properties": {
+                "publicIPAddressVersion": "IPv4",
+                "publicIPAllocationMethod": "Static",
+                "idleTimeoutInMinutes": 4
+            },
+            "copy": {
+                "name": "publicip-loop",
+                "count": 3
+            }
+        },
+        {
+            "type": "Microsoft.Network/virtualNetworks",
+            "apiVersion": "2019-11-01",
+            "name": "[variables('virtualNetworks_myVNet_name')]",
+            "location": "[parameters('location')]",
+            "properties": {
+                "addressSpace": {
+                    "addressPrefixes": [
+                        "[variables('vnet_prefix')]"
+                    ]
+                },
+                "subnets": [
+                    {
+                        "name": "myAGSubnet",
+                        "properties": {
+                            "addressPrefix": "[variables('ag_subnet_prefix')]",
+                            "privateEndpointNetworkPolicies": "Enabled",
+                            "privateLinkServiceNetworkPolicies": "Enabled"
+                        }
+                    },
+                    {
+                        "name": "myBackendSubnet",
+                        "properties": {
+                            "addressPrefix": "[variables('backend_subnet_prefix')]",
+                            "privateEndpointNetworkPolicies": "Enabled",
+                            "privateLinkServiceNetworkPolicies": "Enabled"
+                        }
                     }
-                }
+                ],
+                "enableDdosProtection": false,
+                "enableVmProtection": false
             }
-          ],
-          "frontendPorts": [
-            {
-                "name": "appGatewayFrontendPort",
-                "properties": {
-                    "Port": 80
+        },
+        {
+            "type": "Microsoft.Compute/virtualMachines",
+            "apiVersion": "2019-07-01",
+            "name": "[concat(variables('virtualMachines_myVM_name'), copyIndex(1))]",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/networkInterfaces', concat(variables('net_interface'), copyIndex(1)))]"
+            ],
+            "properties": {
+                "hardwareProfile": {
+                    "vmSize": "[parameters('vmSize')]"
+                },
+                "storageProfile": {
+                    "imageReference": {
+                        "publisher": "MicrosoftWindowsServer",
+                        "offer": "WindowsServer",
+                        "sku": "2016-Datacenter",
```

Comment:
any reason we can't use 2019?

Reply:
this needs to be consistent with my other existing quickstart articles, which use 2016